src/test/regress/expected/multi_partitioning.out

```diff
@@ -2097,21 +2097,21 @@ SELECT citus_shard_cost_by_disk_size(1660207);
 CREATE INDEX idx_btree_hobbies ON "events.Energy Added" USING BTREE ((data->>'location'));
  \c - - - :worker_1_port
 -- should not be zero because of TOAST, vm, fms
-SELECT worker_partitioned_table_size('"events.Energy Added_1660207"');
+SELECT worker_partitioned_table_size('"' ||  relname::text  || '"') FROM pg_class WHERE relname LIKE '%events.Energy Added%' ORDER BY relname LIMIT 1;
```

Comment:
do we still need the string concat operations? Can't we just use `relname` as the parameter to the `worker_partitioned_table_size`?  (for all the tests)

Reply:
alternatively you could probably use `oid` from `pg_class` either directly or by casting with `::regclass`.